### PR TITLE
It's about time - Adds the cheese god (Admin spawnable only)

### PR DESCRIPTION
**Pull request by missatessatessy**
I made this sprite myself about 3 years ago and always planned it to be part of something bigger, life got away from me but I don't want the work I did to be lost on my computer forever. So, I'm adding in the sprite as a god class entity, similar to Honkmother/Ratvar/Narsie, with basic sound and functionality that can be spawned in by admins if they so choose. 

I give permission for anyone who has grand ideas to take this thing further in any way they see fit (even if this PR is closed).

# Document the changes in your pull request

Adds an admin spawnable Cheese God called "The Gouda of All Cheese"
It moves around and randomly spawns different cheese wheels, ghosts can click on it to become a sentient cheese wheel.

I tested it twice and it worked

# Why is this good for the game?
Cheese cult events might be more feasible for admins to do if there's an end goal such as spawning the cheese god.

# Testing
I spawned it, watched it move around and spawn cheese, I clicked on it as a ghost to become a sentient cheese.

# Spriting
gouda

# Wiki Documentation

No wiki documentation yet, but I will add a page to the wiki myself if this is merged.

# Changelog

:cl:  
rscadd: Added cheesegod 
imageadd: added gouda sprite
/:cl:

### CODE REVIEW

**Comment by AMyriad** [comment]
holy shit

File: code/game/objects/structures/cheesegod.dm [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	var/convert_range = 8
+	layer = RIPPLE_LAYER
+	movement_type = PHASING
+	obj_flags = CAN_BE_HIT | DANGEROUS_POSSESSION
+
+/obj/structure/destructible/cheesegod/Initialize(mapload)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+	send_to_playing_players(span_clown("You feel a dairy-like presence.."))
+	sound_to_playing_players('sound/effects/splosh.ogg')
```
Comment (AMyriad) [inline code/game/objects/structures/cheesegod.dm]:
```suggestion
	sound_to_playing_players('sound/effects/splosh.ogg', 100, TRUE)
```
If you want sound to vary

**Review by cuackles — APPROVED** [review]
Good
programmer-15885796

**Comment by LazennG** [comment]
holy fuck

**Comment by Moltijoe** [comment]
Fuckin cheese_act
Cheese heretic path when

**Review by JamieD1 — APPROVED** [review]

**Comment by GentleButter** [comment]
I'm ok with this.

**Pull request merged**